src/WebCrawler.Shared.DevOps/CrawlerBootstrapper.cs

```diff
@@ -19,33 +21,90 @@ namespace WebCrawler.Shared.DevOps
     /// </summary>
     public static class CrawlerBootstrapper
     {
-        public static Akka.Configuration.Config ApplyOpsConfig(this Akka.Configuration.Config previousConfig)
+        /// <summary>
+        ///     Name of the config property used to look for Phobos
+        /// </summary>
+        public const string PhobosEnabled = "phobos-enabled";
+
+        /// <summary>
+        ///     Name of the config property used to direct Phobos' StatsD
+        ///     output.
+        /// </summary>
+        public const string StatsdUrl = "statsd-url";
+
+        /// <summary>
+        ///     Name of the config property used to direct Phobos' StatsD
+        ///     output.
+        /// </summary>
+        public const string StatsdPort = "statsd-port";
+        
+        
+        private static readonly string PetabridgeCmdHocon = @"
+# See petabridge.cmd configuration options here: https://cmd.petabridge.com/articles/install/host-configuration.html
+petabridge.cmd{
+	# default IP address used to listen for incoming petabridge.cmd client connections
+	# should be a safe default as it listens on 'all network interfaces'.
+    host = ""0.0.0.0""
+
+    # default port number used to listen for incoming petabridge.cmd client connections
+    port = 9110
+}";
+        
+        public static ClusterOptions WithOps(this ClusterOptions options)
         {
-            var nextConfig = previousConfig.BootstrapFromDocker();
-            return OpsConfig.GetOpsConfig().ApplyPhobosConfig().WithFallback(nextConfig);
+            // Akka.Cluster split-brain resolver configurations
+            options.SplitBrainResolver = new KeepMajorityOption();
+            
+            return options;
         }
 
-        public static Akka.Configuration.Config ApplyPhobosConfig(this Akka.Configuration.Config previousConfig)
+        public static AkkaConfigurationBuilder WithOps(this AkkaConfigurationBuilder builder, IConfiguration config)
         {
-            var enabledPhobosStr =
-                Environment.GetEnvironmentVariable(OpsConfig.PHOBOS_ENABLED)?.Trim().ToLowerInvariant() ?? "false";
-            if (bool.TryParse(enabledPhobosStr, out var enabledPhobos) && enabledPhobos)
-                return OpsConfig.GetPhobosConfig().WithFallback(previousConfig);
+            builder
+                .AddHocon(config, HoconAddMode.Prepend)
+                .BootstrapFromDocker()
+                .AddHocon(PetabridgeCmdHocon, HoconAddMode.Prepend)
+                .AddPetabridgeCmd(pbm =>
+                {
+                    // enable cluster management commands
+                    pbm.RegisterCommandPalette(ClusterCommands.Instance); 
+                })
+                // Not explicitly setting the liveness provider. The Akka.Remote port
+                // is usually an effective-enough tool for this.
+                .WithHealthCheck(opt =>
+                {
+                    // Use a second socket for TCP readiness checks from K8s
```

Comment:
We need to change how we do things depending on whether we're running in K8s or not - PBM and HealthCheck should not be using the same ports on every process if we're F5-launching from an IDE. I would have a simple config toggle somewhere that just disables these features for all services except maybe WebCrawler (which you can only launch 1 of anyway due to ASP.NET's own binding issues).

Reply:
Might need two different methods:

1. Config when in K8s;
2. Everything else - i.e. uses config-based static address discovery.